collections/Canon_Pat_Lat/MS_Canon_Pat_Lat_52.xml

```diff
@@ -52,173 +45,320 @@
+                           <item>Inceptio tractatus de corpore Christi, folio 65º</item>
+                           <item>Vocabularium paruum, 66 folio</item>
+                           <item>Expositio arboris Danielis spiritualis, 73º folio</item>
+                           <item>Expositio super Aue Maria que dicitur uiridarium uirginis, 79º</item>
+                           <item>Tractatus magistri Bartholomei de luce. Ego sum lux mundi, 100º</item>
+                           <item>Expositio bona et subtilis super Cantica, 105º</item>
+                           <item>Item Expositio uercellensis super Cantica, foliº 131º</item>
+                           <item>Tractatus de laudibus caritatis, 129º</item>
+                           <item>Tractatus Richardi de duodecim patriarchis uel de contemplatione, 147º</item>
+                           <item>Tractatus de 7 gradibus contemplationibus breuis, 166º</item>
+                           <item>Soliloquium Hugonis, folio 167º</item>
+                           <item>Iuibilus Monachorum, 173º</item>
+                           <item>Sermo dominica 5ª post Pascham petite etc., 173º</item>
+                           <item>Postilla super Cantica fratris Egidii, 176</item>
+                           <item>Soliloquium Richardi</item>
+                        </list>
+                     </quote>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
-                     <supportDesc material="perg">
-                        <support>parchment</support>
+                     <supportDesc material="mixed">
+                        <support>parchment and paper (fols. 174–179, 182–187)</support>
+                        <extent>ii (modern paper) + ii (parchment fragment) + 205 + ii (modern paper) leaves <dimensions unit="mm" type="leaf">
+                              <height>180</height>
+                              <width>130</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Contemporary foliation, ‘1‘–‘45‘ (fols. 1v–43r). Modern foliation doubles several numbers: fols. 5a/5b, 23a/23b, 46a/46b, 67a/67b.</foliation>
+                        <collation>a<hi rend="superscript">two</hi> (fols. iii–iv, attached singletons) | 1–3<hi rend="superscript">12</hi> (fols. 1–34, including 5a/5b, 23a/23b) | 4–6<hi rend="superscript">12</hi> (fols. 35–68, including 46a/46b, 67a/67b) | 7<hi rend="superscript">12</hi> (fols. 69–80), 8–9<hi rend="superscript">10</hi> (fols. 81–100) | 10–15<hi rend="superscript">12</hi> (fols. 101–172) | 16<hi rend="superscript">16</hi> (fols. 173–188) | 17<hi rend="superscript">12–1</hi> (fols. 189–199; 12th cancelled)</collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout writtenLines="50 65" columns="2">Ruled in ink, 2 columns of 50–65 lines</layout>
```

Comment:
supply written space?

Reply:
At some point we need to discuss what needs to go into `<layoutDesc>`; I attempted awhile back to create a comparative study of layouts and found both that the way catalogues report written space is incredibly inconsistent, and it's hard to do anything useful with these dimensions if one does not know how it is situated on the page. People such as Muzerelle have suggested instead giving measurements for the positions of guidelines.

Reply:
agreed...